vendor/assets/javascripts/jquery.textile.js

```diff
@@ -74,45 +77,49 @@
       this.$element.attr('rows', 20);
 
       // add Preview to container and hide
-      this.options.$preview = $(this.options.tpl.preview);
       $('.textile-inner', this.options.$wrap).append(this.options.$preview);
       this.options.$preview.hide();
 
       // add Help to container and hide
-      this.options.$help = $(this.options.tpl.help);
       $('.textile-inner', this.options.$wrap).append(this.options.$help)
       this.options.$help.hide();
 
       // toolbar
       this._buildToolbar();
+
+      this.$element.attr('data-textiled', 'true');
     },
     _buildToolbar: function() {
       var button;
 
       // Write
       button = $('<a class="btn-write active" href="javascript:void(null);"><span>Write</span></a>');
-      button.click( $.proxy( function(evt) { this._onBtnWrite(evt); }, this));
+      button.click( function(evt) { this._onBtnWrite(evt) }.bind(this) );
```

Comment:
this change you are doing, from `$.proxy` to `.bind(this)`, I think it is more readable. Is that the reason for the change?

Reply:
They are functionally the same but `$.proxy` relies on jquery while `.bind` is available on plain javascript.